opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -46,20 +46,24 @@ message InstrumentationLibraryMetrics {
 
 // Defines a Metric which has one or more timeseries.
 //
-// The data model and relation between entities is shown in the diagram below.
+// The data model and relation between entities is shown in the
+// diagram below. Here, "DataPoint" is the term used to refer to any
+// one of the specific data point kinds, and "points" is the term use
```

Comment:
"kind" is not defined anywhere. Did you mean Temporality?

Reply:
yeah, wrong copy paste from the other PR. Updated.